models/ArtcilesModel.js

```diff
@@ -51,7 +51,21 @@ const fetchCommentCount = (articleId) => {
           .then((result) => result.rows[0]);
       });
   };
+
+
+  const updateArticleVotes = (article_id, inc_votes) => {
+    return db.query(
+      'UPDATE articles SET votes = votes + $1 WHERE article_id = $2 RETURNING *',
+      [inc_votes, article_id]
+    )
+      .then((result) => {
+        return result.rows[0];
+      })
+      .catch((err) => {
+        throw err; 
```

Comment:
I wouldn't recommend catching in the models or throwing errors, your controller should be able to catch any errors from invoking the model and pass them to your error handling middleware. Throwing an error isn't handling an error